awx/ui_next/src/screens/Inventory/InventoryGroups/InventoryGroupDetail/InventoryGroupDetail.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { GroupsAPI } from '@api';
+import { Route } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+import { act } from 'react-dom/test-utils';
+import { mountWithContexts, waitForElement } from '@testUtils/enzymeHelpers';
+
+import InventoryGroupDetail from './InventoryGroupDetail';
+
+jest.mock('@api');
+const inventoryGroup = {
+  name: 'Foo',
+  description: 'Bar',
+  variables: 'bizz: buzz',
+  id: 1,
+  created: '2019-12-02T15:58:16.276813Z',
+  modified: '2019-12-03T20:33:46.207654Z',
+  summary_fields: {
+    created_by: {
+      username: 'James',
+      id: 13,
+    },
+    modified_by: {
+      username: 'Bond',
+      id: 14,
+    },
+  },
+};
+describe('<InventoryGroupDetail />', () => {
+  let wrapper;
+  let history;
+  beforeEach(async () => {
+    await act(async () => {
+      history = createMemoryHistory({
+        initialEntries: ['/inventories/inventory/1/groups/1/edit'],
```

Comment:
If we are mounting the InventoryGroupDetail then I think the history initial entry should be the path of the inventory group detail: `/inventories/inventory/1/groups/1/detail`